Mathlib/LinearAlgebra/BilinearMap.lean

```diff
@@ -410,6 +410,13 @@ abbrev _root_.Submodule.restrictBilinear (p : Submodule R M) (f : LinearMap.Bili
     LinearMap.BilinForm R p :=
   f.compl₁₂ p.subtype p.subtype
 
+/-- `linMulLin f g` is the bilinear form mapping `x` and `y` to `f x * g y` -/
+def linMulLin (f g : M →ₗ[R] R) : M →ₗ[R] M →ₗ[R] R :=
```

Comment:
This doesn't feel like it belongs in this PR (unless I'm missing something). Can you split it out into a separate PR?